packages/rich-text/src/plugins/Hyperlink/index.tsx

```diff
@@ -63,6 +64,79 @@ export function createHyperlinkPlugin(sdk: FieldExtensionSDK): SlatePlugin {
     pluginKeys: LINK_TYPES,
     inlineTypes: getSlatePluginTypes(LINK_TYPES),
     onKeyDown: buildHyperlinkEventHandler(sdk),
+    deserialize: (editor) => {
+      const hyperlinkOptions = getSlatePluginOptions(editor, INLINES.HYPERLINK);
+      const entryHyperlinkOptions = getSlatePluginOptions(editor, INLINES.ENTRY_HYPERLINK);
+      const assetHyperlinkOptions = getSlatePluginOptions(editor, INLINES.ASSET_HYPERLINK);
+
+      const isAnchor = (element) =>
+        element.nodeName === 'A' &&
+        !!element.getAttribute('href') &&
+        element.getAttribute('href') !== '#';
```

Comment:
What's wrong with that? Is this something that comes up a lot in gdocs `href`s? (I get that the link wouldn't point anywhere -- even so...

I also wonder if we should think about js injection here via `javascript:`. But not something to capture in this PR and I don't think it's something we care about in the legacy editor anyhow.

Reply:
Nothing is wrong to be honest... maybe we should validate if the `href` attribute is a valid URL?